contracts/vaults/YearnVaults/RibbonDeltaYearnVault.sol

```diff
@@ -0,0 +1,357 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.7.3;
+pragma experimental ABIEncoderV2;
+
+import {SafeMath} from "@openzeppelin/contracts/math/SafeMath.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {SafeERC20} from "../../vendor/CustomSafeERC20.sol";
+import {GnosisAuction} from "../../libraries/GnosisAuction.sol";
+import {
+    OptionsDeltaYearnVaultStorage
+} from "../../storage/OptionsVaultYearnStorage.sol";
+import {Vault} from "../../libraries/Vault.sol";
+import {VaultLifecycleYearn} from "../../libraries/VaultLifecycleYearn.sol";
+import {ShareMath} from "../../libraries/ShareMath.sol";
+import {RibbonVault} from "./base/RibbonVault.sol";
+import {IRibbonThetaVault} from "../../interfaces/IRibbonThetaVault.sol";
+
+contract RibbonDeltaYearnVault is RibbonVault, OptionsDeltaYearnVaultStorage {
```

Comment:
Since the Delta vaults don't deal with yvUSDC, why do we need to have a Yearn implementation?

Reply:
I was thinking buying options whilst keeping the reserves in yvUSDC so your net loss every week is smaller. But I guess not worth.